ideascube/library/tests/test_migrations.py

```diff
@@ -0,0 +1,35 @@
+from ideascube.tests.helpers import MigrationTestCase
+
+
+section_int_to_name = {1: 'digital',
+               2: 'children-cartoons',
+               3: 'children-novels',
+               10: 'children-poetry',
+               11: 'children-theatre',
+               4: 'children-documentary',
+               5: 'children-comics',
+               6: 'adults-novels',
+               7: 'adults-documentary',
+               8: 'adults-comics',
+               12: 'adults-poetry',
+               13: 'adults-theatre',
+               9: 'game',
+               99: 'OTHER'}
```

Comment:
You're duplicating the values from the migration file, here. Any way to DRY-ify this a bit?